src/main/target/COLIBRI_RACE/i2c_bst.c

```diff
@@ -372,9 +372,9 @@ static bool bstSlaveProcessFeedbackCommand(uint8_t bstRequest)
             bstWrite16(compassConfig()->mag_declination / 10);
 
             bstWrite8(voltageSensorADCConfig(VOLTAGE_SENSOR_ADC_VBAT)->vbatscale);
-            bstWrite8(batteryConfig()->vbatmincellvoltage);
-            bstWrite8(batteryConfig()->vbatmaxcellvoltage);
-            bstWrite8(batteryConfig()->vbatwarningcellvoltage);
+            bstWrite16(batteryConfig()->vbatmincellvoltage);
```

Comment:
This cannot be changed - the client for this is proprietary software from TBS, and until they change their implementation of the API, changing it unilaterally will break this for all users.